### ISSUE TITLE
Synchronized batchnorm

### ISSUE THREAD

**Issue by aminert** [google-research/simclr#114]
Hi there, can you comment on the importance of synchronized batchnorm for multi-gpu training? What would be the impact of using regular batchnorm instead? Thank you!

**Comment by chentingpc** [comment]
It depends on how many note/gpus do you use, the more the nodes, the easier it is for local BN to leak clues about positive/negatives so the model won't learn as generalizable features as it would with global/synchronized BN.

**Comment by ZihaoH** [comment]
> It depends on how many note/gpus do you use, the more the nodes, the easier it is for local BN to leak clues about positive/negatives so the model won't learn as generalizable features as it would with global/synchronized BN.

Can I understand it as the smaller the batch size, the easier for local bn to leak clues? For an instance of 256 batch size, running on single GPU(equivalent to 8 GPUs using syncBN) is better than 8 GPUs with regular BN?

expecting your reply
Thanks!

**Comment by chentingpc** [comment]
Correct. *The smaller each local batch size per GPU (given the global batch size fixed), the easier it is for the model to cheat.
